src/xunit.console/CommandLine.cs

```diff
@@ -395,6 +396,71 @@ protected XunitProject Parse(Predicate<string> fileExists)
             return project;
         }
 
+        static void IncludeFromFile(XunitProject project, String path)
+        {
+            using (var input = File.OpenText(path))
+            {
+                string line;
+                int linenum = 0;
+                var spaceOrTab = " \t".ToCharArray();
+                while ((line = input.ReadLine()?.Trim()) != null)
```

Comment:
If starting to parsing files,
What do you say about using ReadLineAssync
Meaning changing the upstream flow to Async model as well, starting from the root (Program.Main)
?

Reply:
Changing the application from sync to async may be valuable, but I would consider it way out of scope for the intention of this change.